go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/confluentinc/ccloud-sdk-go-v2/apikeys v0.4.0
 	github.com/confluentinc/ccloud-sdk-go-v2/billing v0.3.0
 	github.com/confluentinc/ccloud-sdk-go-v2/byok v0.0.2
-	github.com/confluentinc/ccloud-sdk-go-v2/ccl v0.2.0
+	github.com/confluentinc/ccloud-sdk-go-v2/ccl v0.3.0
```

Comment:
I think we'll need to release a new version of the ccl SDK that doesn't require go 1.22.8. Updating to 1.22.8 currently blocks releases.

Reply:
downgrade it to 1.22.7